### PR TITLE
Generate documenation around "Monetary Posts"

### PR DESCRIPTION
**Pull request by arjunrbery**
This documentation will cover the best practices around mutual aid, charity fundraisers, affiliate links, and crowdfunding.

### CODE REVIEW

File: community.hachyderm.io/content/en/docs/monetary-posts/_index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+All others user posts related to charity fundraisers should include "This post is for `#FediFundraising`" either at the beginning or end of your post.
+
+## Affiliate Links
+
+### Description
+
+Affiliate links allow an individual to earn a commission when someone purchases a product or service through their link. Affiliate links are a way for you to earn money without having to create a product or service.
+
+Typically users that are posting affiliate links are doing so from a business capacity. As a result, we ask that ver
```
Comment (ThisIsMissEm) [inline community.hachyderm.io/content/en/docs/monetary-posts/_index.md:62]: This sentence is incomplete, fyi

File: community.hachyderm.io/config.toml
Comment (freiheit) [inline community.hachyderm.io/config.toml:1]: config.toml changes accidentally included in the PR?

File: community.hachyderm.io/content/en/docs/monetary-posts/_index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+Crowdfunding is a way for individuals to raise money for a specific project or cause. Crowdfunding is typically done through a platform such as Kickstarter, Indiegogo, or GoFundMe.
+
+### Posting about Crowdfunding
+
+Please use the below matrix to determine how to interact with crowdfunding posts.
+
+| Type of Crowdfunding | Poster Involved | Poster is not Involved |
+| --- | --- | --- |
+| Mutual Aid | Crowdfunding for a mutual aid effort is allowed. Please review the [mutual aid](#mutual-aid) section above. | Please ensure that the individual(s) impacted are okay with the distribution of their information. |
+| Not Mutual Aid | As a community that focuses on science and technology, we want to support the incredible work that is being done by our users. However, we ask that you be mindful of the frequency of your posts and review the [Specialized Account Expectations](/docs/account-types/covenant/) for more information. For non-tech crowdfunding, we recommend an alternative platform. | We are excited that you are excited! |
```
Comment (freiheit) [inline community.hachyderm.io/content/en/docs/monetary-posts/_index.md:84]: I think this should explicitly require disclosure if the poster has a financial interest (their project, their employer).

File: community.hachyderm.io/config.toml
Comment (arjunrbery) [reply community.hachyderm.io/config.toml:1]: The config.toml enables search to make it easier to find content in docs.

**Pull request merged**